src/test/resources/xades4j/production/SignerSpecificTest.properties

```diff
@@ -0,0 +1 @@
+common.name=\u041E\u041E\u041E \u041E\u0440\u0433\u0430\u043D\u0438\u0437\u0430\u0446\u0438\u044F
```

Comment:
Could this be written as a string literal in the test to avoid the additional file?

Reply:
I removed additional file.